HTTPlayground/views.py

```diff
@@ -0,0 +1,49 @@
+from cgi import FieldStorage
+from jinja2 import Template
+from HTTPlayground.base import FileReader, CookieHandler, send_headers
+
+
+def login(request):
+    cookie_header = {'Set-Cookie': CookieHandler.generate_cookie(request, {'auth': 'True'})}
+    send_headers(request, content_type='text/html', **cookie_header)
+    t = Template(FileReader.read('login.html'))
+    rendered_html = t.render(link='/form')
+    return rendered_html
+
+
+def logout(request):
+    cookie_header = {'Set-Cookie': CookieHandler.generate_cookie(request, {'auth': 'False'})}
```

Comment:
I'd add smth like `CookieHandler.remove_cookie(request, name)` instead of setting it to `False`